tests/antithesis/Makefile

```diff
@@ -75,3 +76,23 @@ validate-node-count:
 		echo "CFG_NODE_COUNT must be either 1 or 3 (got $(CFG_NODE_COUNT))"; \
 		exit 1; \
 	fi
+
+REQUIRED_K8S_TOOLS := kubectl kind kapp
+
+.PHONY: check-k8s-tools
+check-k8s-tools:
+	@for tool in $(REQUIRED_K8S_TOOLS); do \
+		if ! command -v $$tool >/dev/null 2>&1; then \
+			echo "Error: '$$tool' is missing.  Please install it before continuing." >&2; \
+			exit 1; \
+		fi \
+	done
+
+.PHONY: antithesis-create-local-k8s-cluster
+antithesis-create-local-k8s-cluster: check-k8s-tools
+	kind create cluster || true
+
+.PHONY: antithesis-deploy-local-k8s-setup
+antithesis-deploy-local-k8s-setup: antithesis-create-local-k8s-cluster antithesis-build-etcd-image-main
```

Comment:
While it's good to combine targets during compilation, in this case we don't benefit hugely from having 1 instead of 2 or 3 commands. We can just put them in readme. It's better to give users control instead of forcing into 1 path.

Reply:
Agree with this point.  I'll remove the targets for now and add the instructions instead.